toolchest/rpm/utils.py

```diff
@@ -211,6 +211,21 @@ def dlrn_label_compare(l, r):
     return (rpmLabelCompare((lx, lv, lr), (rx, rv, rr)), altered)
 
 
+def cpaas_label_compare(l, r):
```

Comment:
I would suggest changing 'l' and 'r' to be 'left' and 'right'.  I recently ran autopep on this, and we have a ton of changes needed to get that compliant, largely due to it calling these 'ambiguous variables'.  I am not asking for the existing stuff to be changed, we will do a patch (or patches) just for that soon, but I would prefer not to introduce more warnings. The same error is almost surely going to be trigger in the test.

I recently (temporarily) disabled this in our flake8 testing, but if you want to run it yourself locally, you will see what I mean.

Reply:
Fair point. Will do.